batch/batch/server/server.py

```diff
@@ -158,7 +195,7 @@ def delete(self):
         if self.batch_id:
             batch = batch_id_batch[self.batch_id]
             batch.remove(self)
-
+        self.notify_children('Cancelled')
```

Comment:
Might be better to set state to Cancelled?  Also delete pod then notify for pleasing symmetry with `cancel`.

Reply:
addressed